src/main/resources/templates/desktop/page.twig

```diff
@@ -3,6 +3,7 @@
 <head>
 	<title>{{ title }}</title>
 	<link rel="stylesheet" type="text/css" href="/res/desktop.css?{{ staticHashes['desktop.css'] }}"/>
+    <link rel="stylesheet" type="text/css" href="/res/audioplayer_desktop.css?{{ staticHashes['audioplayer_desktop.css'] }}">
```

Comment:
Why make it a separate file and cause an additional request? Just include it into desktop.css/mobile.css, it's tiny enough as it is and will compress to almost nothing

Reply:
Because I also include it inline in embedded posts. I don't think it's a good idea to inline the entirety of desktop.css in embedded posts, and I couldn't think of a better way to handle this.

Reply:
You can `@include` it in desktop.css, no?

Reply:
Although now, I think I know a better way.

Reply:
Fixed